### PR TITLE
 Add FieldPropsManager argument to TransMult constructors

### PR DESCRIPTION
**Pull request by joakim-hove**
Enable testing of the new 3D properties in the Transmult code

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp [hunk 1]
```diff
@@ -97,13 +97,19 @@ namespace Opm {
...
+        m_e3DProps(e3DProps) {
 
+#ifdef ENABLE_3DPROPS_TESTING
         for (size_t idx = 0; idx < keywords.size(); idx++)
             this->addKeyword(e3DProps, *keywords[idx] , e3DProps.getDefaultRegionKeyword());
-
+#else
+        for (size_t idx = 0; idx < keywords.size(); idx++)
+            this->addKeyword(e3DProps, *keywords[idx] , e3DProps.getDefaultRegionKeyword());
+#endif
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp]: Is there supposed to be a difference between these two cases (`ENABLE` and `!ENABLE`)?  I don't see a difference here.

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp]: Yes - they should be different; but this is still *very much* work in progress!

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by akva2** [comment]
.... and norne init is 15x slower again.

**Comment by joakim-hove** [comment]
I have made a revert PR here: https://github.com/OPM/opm-common/pull/1283

I will merge it when it is green - or you can do it right away.

**Comment by joakim-hove** [comment]
Before the revert - are you positive this PR is the offending? It seems a bit unlogical to me

**Comment by joakim-hove** [comment]
Before the revert - are you positive this PR is the offending? It seems a bit unlogical to me.

**Comment by joakim-hove** [comment]
OK - I have looked briefly on this; so far not really clear to me. Since it is weekend now, and I don't fully understand the issue I will let this linger now. If anyone is bitten by this - and have a fix - before monday - **merge away!**

Sorry :-(

**Comment by joakim-hove** [comment]
> .... and norne init is 15x slower again.

You were right - this was the culprit. Temp "fix" here: #1284

**Pull request merged**